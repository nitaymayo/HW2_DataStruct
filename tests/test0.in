add_horse 1 100
add_horse 2 200
add_horse 3 300
add_herd 1
join_herd 1 1
join_herd 2 1
join_herd 3 1
follow 1 2
can_run_together 1
follow 2 3
can_run_together 1
leave_herd 3
join_herd 3 1
can_run_together 1
follow 2 1
can_run_together 1
follow 2 3
can_run_together 1
leads 1 3
leads 2 1
get_speed 1
get_speed 2
get_speed 3