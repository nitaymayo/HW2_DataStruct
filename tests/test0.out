add_horse: SUCCESS
add_horse: SUCCESS
add_horse: SUCCESS
add_herd: SUCCESS
join_herd: SUCCESS
join_herd: SUCCESS
join_herd: SUCCESS
follow: SUCCESS
can_run_together: SUCCESS, False
follow: SUCCESS
can_run_together: SUCCESS, True
leave_herd: SUCCESS
join_herd: SUCCESS
can_run_together: SUCCESS, False
follow: SUCCESS
can_run_together: SUCCESS, False
follow: SUCCESS
can_run_together: SUCCESS, True
leads: SUCCESS, True
leads: SUCCESS, False
get_speed: SUCCESS, 100
get_speed: SUCCESS, 200
get_speed: SUCCESS, 300
